### ISSUE TITLE
Bot stops sending messages back and forth

### ISSUE THREAD

**Issue by moerkoetsmeuk** [manuelgu/DiscordMC#43]
Hi there,
Hopefully this is still the right place to submit , and you're still working on the bot.
Since it's not been updated to the latest versions I'm also unsure if it can still live up to its fullest expectations.

Anyhow, on my server (1.14.4, then 1.15.2) I've been having the problem where the bot just stops responding after a while. Like at first it works just fine, but after an hour or so it stops sending messages back and forth. Also no join/leave/death messages. When I use /discord login it says that it's already online. (it does still show as on online in Discord).
Here is the output from the debug command if it helps: https://hastebin.com/olidogadep

**Comment by manuelgu** [comment]
Indeed, this has not been updated in a while. There have been several changes to the Discord API which are not reflected in the library that is being used here. Unfortunately, I have no time to update the plugin to a newer version of a Discord library.

**Comment by moerkoetsmeuk** [comment]
Hey, thanks for the response. That's totally understandable, real life comes first. We'll hear it if you ever want to make a come back. Take care!
